public/static/docs/command-reference/gc.md

```diff
@@ -14,17 +15,25 @@ usage: dvc gc [-h] [-q | -v] [-a] [-T] [-c] [-r <name>]
 This command deletes (garbage collects) data files or directories that may exist
 in the cache (or [remote storage](/doc/command-reference/remote) if `-c` is
 used) but no longer referenced in [DVC-files](/doc/user-guide/dvc-file-format)
-currently in the <abbr>workspace</abbr>. By default, this command only cleans up
-the local cache, which is typically located on the same machine as the project
-in question. This usually helps to free up disk space.
+currently in the <abbr>workspace</abbr>. To avoid accidentally deleting data,
+this command requires the explicit use of [option](#options) flags to determine
+it's behavior (i.e. what "garbage" to collect).
 
-There are important things to note when using Git to version the
-<abbr>project</abbr>:
+By default, this command won't delete anything at all to make it safe and
```

Comment:
looks like a repetition to the previous sentence

Reply:
fixed, thanks